docs/build/tutorials/platform/adjusting-gas-price-during-high-network-activity.md

```diff
@@ -0,0 +1,105 @@
+| ----------- | ----------- | ----------- | ---------- | ------------- | ----------- |
+| 1           | 50 GWEI     | 0 GWEI      | 25 GWEI    | 0 GWEI        | 25 GWEI     |
+| 2           | 50 GWEI     | 0 GWEI      | 50 GWEI    | 0 GWEI        | 50 GWEI     |
+| 3           | 50 GWEI     | 0 GWEI      | 60 GWEI    | 0 GWEI        | **PENDING**|
+| A           | 50 GWEI     | 10 GWEI     | 40 GWEI    | 10 GWEI       | 50 GWEI     |
+| B           | 40 GWEI     | 40 GWEI     | 40 GWEI    | 0 GWEI        | 40 GWEI     |
+
+Look at transactions **A** and **B**. In this scenario, it looks like transaction B is paying a higher tip, however, this depends on the base fee of the block where the transactions are included. The effective tip of A is more than that of B. So, if both of these transaction competes for being included in the next block, then the validators would prioritize transaction A, due to higher effective tip.
+
+
+## Why my transaction is on hold or failing?
+
+If your transaction is failing and giving an error - `transaction underpriced`, then the max fee cap of your transaction must be less than the minimum base price that the network supports (as of now, it's 25 nAVAX or GWEI). Although the base fee is automatically estimated in wallets like Metamask, you can try increasing the max fee cap in the wallet.
+
+During a period of heavy congestion on the network, all submitted transactions couldn't be included in the same block, due to the block's gas limit. So, validators choose transactions giving more priority to transactions with higher effective tip. Your transaction will have to wait until the effective tip is highest among the pending transactions.
+
+Another reason for pending transactions is the max fee cap being significantly below the current base fee that the network is charging. In this case, we need to increase the max fee cap of the transaction.
+
+These fee adjustments can be made through wallets like Metamask.
+
+## Adjusting gas fees before submitting the transaction
+
+You may not need to edit the gas fees on normal days. This is only required if there is heavy congestion on the network, and the base fees are frequently fluctuating.
+
+1. Let's create a sample transaction on Fuji testnet, in which we will be sending 0.1 AVAX to a receiver using the Metamask wallet extension. By clicking **Next** we can review gas fees and the amount which we want to send.
+
+![](/img/dynamic-fees-adjustment-1.png)
+
+
+2. On the review page, you can see the estimated priority and max fee for this transaction. Now click on **EDIT**, to adjust these fees according to network requirements.
+
+![](/img/dynamic-fees-adjustment-2.png)
+
+
+3. On this page, you can edit the priority fee (gas tip cap) and max fee (max fee cap). You can estimate the max fee as shown on [snowtrace](https://snowtrace.io/gastracker) which represents the average max fee over the last 3 seconds.
```

Comment:
I would suggest also adding reference to https://stats.avax.network/dashboard/c-chain-activity/ for current gas. The snowtrace gastracker is not always correct.